### ISSUE TITLE
Some pause screen controls lack visible focus indicators

### ISSUE THREAD

**Issue by domstoppable** [ChaMP-Lab/SurvivalShooter#15]
When navigating the pause screen with a controller, the sliders checkboxes do not provide any visual indication that they have focus. They can modified via gamepad, but the user has no way of knowing which widget they are on until they try to change its value
